slash/plugins/builtin/xunit.py

```diff
@@ -48,10 +48,18 @@ def error_added(self, result, error):
 
     def _add_error(self, errortype):
         exc_type, exc_value, exc_tb = exc_info = sys.exc_info()
+        self._add_test_metadata(errortype,
+                                dict(type=exc_type.__name__,
+                                     message=str(exc_value)),
+                                get_traceback_string(exc_info))
+
+    def _add_test_metadata(self, title, attributes=None, content=None):
```

Comment:
This shouldn't start with an underscore, as it implies being a private method. You should use a regular method, specifying it is not registered on any hook like in http://slash.readthedocs.org/en/master/plugins.html#plugin-activation